clientconn_test.go

```diff
@@ -689,3 +689,16 @@ func TestDisableServiceConfigOption(t *testing.T) {
 		t.Fatalf("want: method (\"/foo/bar/\") config to be empty, got: %v", m)
 	}
 }
+
+func TestGetClientConnTarget(t *testing.T) {
+	r, cleanup := manual.GenerateAndRegisterManualResolver()
```

Comment:
Resolver is not necessary in this test.
Just `Dial("nonexist:///non.existent")` would work.